9_Find_Pythagorean_Triple.py

```diff
@@ -0,0 +1,19 @@
+
+
+def find_triple_given_sum(sum):
+    a = 1
+    while a <= sum/3:
+
+        b = a + 1
+        while b <= sum/2:
+
+            c = sum - a - b
+            print(c)
+            if ( a*a + b*b == c*c):
```

Comment:
It would be good practice to split the "is indeed a pythagorean triple" check into a separate function